bin/boot

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env sh
 set -ex
 bin/setup
-bundle exec foreman start
+bundle exec foreman start 2>&1 | cut -d '|' -f 2
```

Comment:
I did not find any way to modify/remove the log output of `bundle`, so I `cut` the output